makefile

```diff
@@ -26,9 +26,9 @@ docker:
 	    --build-arg https_proxy \
 		-f Dockerfile \
 		--label "git_sha=$(GIT_SHA)" \
-		-t edgexfoundry/docker-app-service-configurable:$(GIT_SHA) \
-		-t edgexfoundry/docker-app-service-configurable:master-dev \
-		-t nexus3.edgexfoundry.org:10004/docker-app-service-configurable:master-dev \
+		-t edgexfoundry/app-service-configurable:$(GIT_SHA) \
+		-t edgexfoundry/app-service-configurable:${VERSION}-dev \
+		-t nexus3.edgexfoundry.org:10004/app-service-configurable:${VERSION}-dev \
```

Comment:
should ${VERSION} be ${APPVERSION} ; it doesn't work for me when I tried to `make docker` locally if using VERSION

Reply:
all in good time and taken care of. thanks @lenny-intel !